misc/vtk/include/inviwo/vtk/util/vtkutil.h

```diff
@@ -55,5 +55,34 @@ inline dvec3 extentFromBounds(const double* bounds) {
 inline dvec3 offsetFromBounds(const double* bounds) {
```

Comment:
This feels a bit unsafe. Should probably document the assumption on type of VTK-bounds. Why inline?
You might want to export these functions (IVW_MODULE_VTK_API) so that dependent modules can use the functionality

Reply:
This code has nothing to do with the pull request, was neither used nor changed.